ipyleaflet/leaflet.py

```diff
@@ -1038,8 +1039,8 @@ def _default_layers(self):
 
         return (basemap,)
 
-    bounds = Tuple(read_only=True)
-    bounds_polygon = Tuple(read_only=True)
+    bounds = Tuple(read_only=False)
+    bounds_polygon = Tuple(read_only=False)
```

Comment:
You can't change this, those two parameters needs to remain `read_only`

Reply:
Part of my experiments, going to revert it.